################################################################################
SHORT DESCRIPTION: 
################################################################################

################################################################################
LONG DESCRIPTION: 
################################################################################

################################################################################
CHANGELOG 
################################################################################

################################################################################
REMOVAL 
################################################################################

################################################################################
UPSTREAM PLAN
################################################################################

################################################################################
INTERNAL DEPENDENCIES 
################################################################################

################################################################################
PATCHES 
################################################################################
Index: xen-4.6.5/xen/xsm/flask/hooks.c
===================================================================
--- xen-4.6.5.orig/xen/xsm/flask/hooks.c
+++ xen-4.6.5/xen/xsm/flask/hooks.c
@@ -738,6 +738,9 @@ static int flask_domctl(struct domain *d
     case XEN_DOMCTL_psr_cat_op:
         return current_has_perm(d, SECCLASS_DOMAIN2, DOMAIN2__PSR_CAT_OP);
 
+    case XEN_DOMCTL_setcorespersocket:
+        return current_has_perm(d, SECCLASS_DOMAIN2, DOMAIN2__SETCORESPERSOCKET);
+
     default:
         printk("flask_domctl: Unknown op %d\n", cmd);
         return -EPERM;
Index: xen-4.6.5/xen/xsm/flask/policy/access_vectors
===================================================================
--- xen-4.6.5.orig/xen/xsm/flask/policy/access_vectors
+++ xen-4.6.5/xen/xsm/flask/policy/access_vectors
@@ -216,6 +216,8 @@ class domain2
     setscheduler
 # XENMEM_claim_pages
     setclaim
+# XEN_DOMCTL_setcorespersocket
+    setcorespersocket
 # XEN_DOMCTL_set_max_evtchn
     set_max_evtchn
 # XEN_DOMCTL_cacheflush
